Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["mopro-core", "mopro-ffi", "mopro-cli", "ark-zkey"]
+members = ["witness", "mopro-core", "mopro-ffi", "mopro-cli", "ark-zkey"]
```

Comment:
I understand this is a draft research, but is the goal to embed this in repo? Or to point to upstream/forked version?